drivers/overlay/overlay.go

```diff
@@ -1200,8 +1252,26 @@ func (d *Driver) ApplyDiff(id, parent string, options graphdriver.ApplyDiffOpts)
 
 func (d *Driver) getDiffPath(id string) string {
 	dir := d.dir(id)
+	diffPath := path.Join(dir, "diff")
+	if ld, err := os.Readlink(diffPath); err == nil {
```

Comment:
how would a relative symlink work in this case?  Do we need to raise an error if the symlink is not absolute?

Reply:
fixed to allow only absolute paths.